src/Compilers/Core/Portable/StrongName/CryptoBlobParser.cs

```diff
@@ -181,62 +190,72 @@ private static ImmutableArray<byte> CreateSnPublicKeyBlob(byte type, byte versio
         /// <remarks>
         /// Can be either a PUBLICKEYBLOB or PRIVATEKEYBLOB. The BLOB must be unencrypted.
         /// </remarks>
-        public unsafe static bool TryGetPublicKey(ImmutableArray<byte> blob, out ImmutableArray<byte> publicKey)
+        public unsafe static bool TryParseKey(ImmutableArray<byte> blob, out ImmutableArray<byte> snKey, out RSAParameters? privateKey)
```

Comment:
That's true.  I'll remove this.  Also, if it's a priority, I can likely rewrite IsValidPublicKeyUnsafe to be safe too.